pkg/networkmanager/firewall.jsx

```diff
@@ -760,7 +761,24 @@ class ActivateZoneModal extends React.Component {
                         <FormHelperText isHidden={false}>{_("The cockpit service is automatically included")}</FormHelperText>
                     </FormGroup>
 
-                    <FormGroup label={ _("Interfaces") } hasNoPaddingTop isInline>
+                    <FormGroup label={ _("Interfaces") }
+                               hasNoPaddingTop
+                               isInline
+                               labelIcon={
+                                   <Popover aria-label={_("Inerfaces help")}
+                                     enableFlip
+                                     bodyContent={
+                                         <Flex direction={{ default: 'column' }}>
+                                             <FlexItem>{_("Traffic passing through selected interfaces will respect this zone's settings.")}</FlexItem>
+                                             <FlexItem>{_("An interface cannot belong to multiple zones.")}</FlexItem>
```

Comment:
I found "to only one zone" easier to understand, but no strong opinion. As this was vetted by Garrett and Lis already, fine for me.

Again, it feels a bit pointless to me to put that into a help text. It only aggravates the uncertainty of the user -- "So do I need to make sure that it's unique here? Why doesn't the UI enforce that?" *Do* we enforce that in firewalld/the UI actually? I should not be able to select the same interface in more than one zone, and then we wouldn't need to explain it either.

Reply:
You cannot select interface from UI that is already assigned to different zone.

Reply:
*phew*, that's good. Thanks!